lib/msf/core/exploit.rb

```diff
@@ -1193,9 +1193,23 @@ def handler_enabled?
   # value can be one of the Handler::constants.
   #
   def handler(*args)
-    return if not payload_instance
-    return if not handler_enabled?
-    return payload_instance.handler(*args)
+    return unless (
```

Comment:
return unless is preferable here, but please do not compound them. keep them as two separate one-line conditionals. this is both easier to read and to debug